dist/husky.css

```diff
@@ -10562,22 +10562,29 @@ a.btn-black, a.btn-highlight {
   width: 100%;
   height: 70px;
   position: relative;
-  top: 0px;
-  left: 0px;
+  top: 0;
+  left: 0;
   color: #fff;
-  line-height: 70px;
   padding-left: 55px;
   font-size: 12px;
-  z-index: 15000;
+  z-index: 200;
```

Comment:
you totally right the z-indexes are widely distributed in the project. whenever you change something it could make problems anywhere. but i have no idea where it could make problems (: we will see!